client/flutter/lib/pages/license_page.dart

```diff
@@ -1,481 +1,28 @@
-                      Container(height: 18.0),
-                      Text(applicationLegalese ?? '', style: Theme.of(context).textTheme.caption),
-                    ],
-                  ),
-                ),
-              ),
-            ],
-          ),
-          ...?children,
-        ],
-      ),
-      actions: <Widget>[
-        FlatButton(
-          child: Text(MaterialLocalizations.of(context).viewLicensesButtonLabel),
-          onPressed: () {
-            showLicensePage(
-              context: context,
-              applicationName: applicationName,
-              applicationVersion: applicationVersion,
-              applicationIcon: applicationIcon,
-              applicationLegalese: applicationLegalese,
-            );
-          },
-        ),
-        FlatButton(
-          child: Text(MaterialLocalizations.of(context).closeButtonLabel),
-          onPressed: () {
-            Navigator.pop(context);
-          },
-        ),
-      ],
-      scrollable: true,
-    );
-  }
-}
-
-/// A page that shows licenses for software used by the application.
-///
-/// To show a [LicensePage], use [showLicensePage].
-///
-/// The [AboutDialog] shown by [showAboutDialog] and [AboutListTile] includes
-/// a button that calls [showLicensePage].
-///
-/// The licenses shown on the [LicensePage] are those returned by the
-/// [LicenseRegistry] API, which can be used to add more licenses to the list.
-class LicensePage extends StatefulWidget {
-  /// Creates a page that shows licenses for software used by the application.
-  ///
-  /// The arguments are all optional. The application name, if omitted, will be
-  /// derived from the nearest [Title] widget. The version and legalese values
-  /// default to the empty string.
-  ///
-  /// The licenses shown on the [LicensePage] are those returned by the
-  /// [LicenseRegistry] API, which can be used to add more licenses to the list.
-  const LicensePage({
-    Key key,
-    this.applicationName,
-    this.applicationVersion,
-    this.applicationIcon,
-    this.applicationLegalese,
-  }) : super(key: key);
-
-  /// The name of the application.
-  ///
-  /// Defaults to the value of [Title.title], if a [Title] widget can be found.
-  /// Otherwise, defaults to [Platform.resolvedExecutable].
-  final String applicationName;
-
-  /// The version of this build of the application.
-  ///
-  /// This string is shown under the application name.
-  ///
-  /// Defaults to the empty string.
-  final String applicationVersion;
-
-  /// The icon to show below the application name.
-  ///
-  /// By default no icon is shown.
-  ///
-  /// Typically this will be an [ImageIcon] widget. It should honor the
-  /// [IconTheme]'s [IconThemeData.size].
-  final Widget applicationIcon;
-
-  /// A string to show in small print.
-  ///
-  /// Typically this is a copyright notice.
-  ///
-  /// Defaults to the empty string.
-  final String applicationLegalese;
+class WHOLicensePage extends StatefulWidget {
```

Comment:
No need for WHO prefix.

Reply:
LicensePage already exists in the material library so it wouldn't work while material was imported.

Fixed by renaming to LicensePage and removing the unnecessary material import.